TaylorFramework/Modules/scissors/ExtendedComponent.swift

```diff
@@ -10,19 +10,23 @@ import Foundation
 import SourceKittenFramework
 import SwiftXPC
 
+typealias Names = (name: String?, typeName: String?)
+
 final class ExtendedComponent {
     
     var offsetRange: OffsetRange
     var type: ComponentType
     var name: String?
```

Comment:
Once we have `var name` and `var typeName`, we can replace them with a `Names` tuple.